api/models/release.go

```diff
@@ -488,6 +489,20 @@ func (r *Release) resolveLinks(app App, manifest *manifest.Manifest) (*manifest.
 
 			port := other.Ports[0]
 
+			if other.Exports["LINK_PORT"] != "" {
+				i, _ := strconv.Atoi(other.Exports["LINK_PORT"])
+				if err != nil {
+					// handle error
+					fmt.Println(err)
```

Comment:
Golang noobie.. not sure what to do here in case of an error converting `LINK_PORT` to an `int`.